PinInput.js

```diff
@@ -49,29 +109,46 @@ class PinInput extends Component {
     const {
       containerDefaultStyle,
       pinDefaultStyle,
-      pinActiveDefaultStyle
+      pinActiveDefaultStyle,
     } = styles;
     /** Props */
     const {
       numberOfPinsActive,
       numberOfPins,
+      maxNumberOfLargePins,
```

Comment:
Let's change this to `numberOfPinsMaximum` just to follow the same naming convention as the two props above. This isn't a strict naming convention for all projects, but it will just create a bit more symmetry in this project.